scopes/html/html/html-docs-app.ts

```diff
@@ -1,15 +1,23 @@
 import mapObject from 'map-obj';
-import DocsRoot, { ReactDocsRoot } from '@teambit/react.ui.docs-app';
+import { DocsRootProps } from '@teambit/docs';
+import DocsRoot from '@teambit/react.ui.docs-app';
 import { htmlToReact } from './html-to-react';
 
-const htmlDocsRoot: ReactDocsRoot = function(...args) {
-  const [Provider, componentId, docs, compositions, ...rest] = args;
+const htmlDocsRoot = function({
+  Provider,
+  componentId,
+  docs,
+  compositions,
+  ...rest
+}: DocsRootProps) {
 
   // should be mapObject<Record<string, any>, Record<string, () => any>>
   // @ts-ignore TODO fix
   const reactCompositions = mapObject(compositions, (key, value) => [key, htmlToReact(value as HTMLElement)]);
 
-  return DocsRoot(Provider, componentId, docs, reactCompositions, ...rest);
+  return DocsRoot({Provider, componentId, docs, compositions: reactCompositions, ...rest});
 }
```

Comment:
you can now forward all of the props (remove Provider, componendId, docs, and pass them through the ...rest)

Reply:
I can, but it's less obvious here what props are being passed and I think it's useful for someone seeing this code to know what's being passed on

Reply:
(seeing as we cant just pass them all through as the compositions needs to be refactored here before forwarding)

Reply:
update - implemented the change